src/styles/blocks/page.scss

```diff
@@ -1,7 +1,11 @@
+* {
+  box-sizing: border-box;
+}
+
 .page {
   font-family: Roboto, sans-serif;
 
-  &__body {
+  .body {
```

Comment:
It is redundant to nest class for body in class for html.
Also not necessary to make class for html tag. It will be always just one on the page.